server/server.js

```diff
@@ -3,7 +3,7 @@ const db = require('../database/index.js');
 
 const App = express();
 
-App.use(express.static('./public/dist'));
+App.use('/:productId', express.static('./public/dist'));
```

Comment:
@danielharmon Is this necessary now that you have brought in React Router? Is it possible to use React Router to display a page according to the product number in the url? Idk the answers to these questions, I am just curious.

Reply:
I think that react router could be used for that on the front end, but on the server side of things it still needs to get a request in with some parameters on it.